### DISCUSSION TITLE
NexoPOS 4.7.2

### DISCUSSION THREAD

**Discussion by Blair2004** [Blair2004/NexoPOS#688] [Releases]
This update of NexoPOS 4.7.x brings mostly package update (node.js and composer). We've also fixed few issues that has been reported by one of our user @osvaldoclasico for which we say thank you. 

# What We're Working On

For the upcoming major upgrade NexoPOS 4.8.x we'll make sure to update Laravel to 9, so that the fact php 8.1 is required for installing NexoPOS 4.7 is justified. 

# Changelog:
- Added: new exception handler
- Added: new doctor method (fix users attributes)
- Fixed: Wrong customer phone validation
- Package update

<hr /><em>This discussion was created from the release <a href='https://github.com/Blair2004/NexoPOS-4x/releases/tag/v4.7.2'>NexoPOS 4.7.2</a>.</em>

**Comment by osvaldoclasico** [comment]
Thank you very much, I tried and it worked fine.
